tools/src/main/scala/scala/scalanative/build/NativeLib.scala

```diff
@@ -182,18 +187,23 @@ private[scalanative] object NativeLib {
   private def copyNativeDir(nativelib: NativeLib): Path = {
     val target = nativelib.dest
     val source = nativelib.src
-    val files = IO.getAll(source, allFilesPattern(source))
-    val fileshash = IO.sha1files(files)
-    val fileshashPath = target.resolve("fileshash")
-    def copied =
-      Files.exists(target) &&
-        Files.exists(fileshashPath) &&
-        Arrays.equals(fileshash, Files.readAllBytes(fileshashPath))
-    if (!copied) {
+    def copy() = {
       IO.deleteRecursive(target)
       IO.copyDirectory(source, target)
-      IO.write(fileshashPath, fileshash)
     }
+    if (Platform.isJVM) {
```

Comment:
I thought I could work on this part using [Kirill A. Korinsky's](https://github.com/catap) work after this gets merged. Not sure about `java.security.DigestInputStream` - I think we can work around that or have a Scala Native internal one not in the java namespace.